draft-ietf-httpbis-client-hints.md

```diff
@@ -266,6 +267,11 @@ Implementers ought to consider both user and server controlled mechanisms and po
 Implementers SHOULD support Client Hints opt-in mechanisms and MUST clear persisted opt-in preferences when any one of site data, browsing history, browsing cache, or similar, are cleared.
 
 
+## Backend Security
+Adding new request headers increases the risk that legacy server systems already use those values for a different purpose. Changing the request header values such legacy systems receive may result in server bugs.
```

Comment:
Hmm, I believe a big part of the motivation for `Sec` prefix is that it's a reserved prefix that only the browser can emit, which seems like something worth mentioning here? This allows the servers to treat such headers with different policies; have some confidence in the data being delivered.


> Adding new request headers introduces potential deployment risks. For example, some servers may already use the same header for a different purpose, which may result in conflicts or other undesired behaviors.
>
>While this risk is significantly mitigated by the opt-in mechanisms of Client Hints, specifications and features relying on Client Hints should use the `Sec-` prefix for request header names. User agents reserve this namespace for headers that can only be emitted by the browser and thus can guarantee and enforce the type and format of the data communicated in the header value fields.

wdyt, warmer?

Reply:
Yup, SGTM